changelog.json

```diff
@@ -1,5 +1,36 @@
 {
   "releases": {
+    "3.2.1": [
+      "[New] Get notified of comments to your pull requests - #16226",
```

Comment:
```suggestion
```

I believe this is still beta feature flagged.
![CleanShot 2023-04-04 at 13 50 40](https://user-images.githubusercontent.com/75402236/229876216-7f0658d2-4048-4d00-8420-b082a97e3880.png)